samples/popup.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Popup Demo</title>
+    <meta name="description" content="Popup demo" />
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+    <script src="chair.js"></script>
+    <script>
+      // TODO: Verify room is not needed and delete it
+      AFRAME.registerComponent("read-room", {
```

Comment:
Can we move the component registration in a diff file? In the actual repo structure all of these will go in `public/static/js/components/*`